### PR TITLE
Reconfigure on_initialize weight.

### PR DESCRIPTION
**Pull request by kianenigma**
This PR tweaks two small values on both polkadot and kusama: 

1. **Minimum-Multiplier**. This is the smallest value of the weight to fee multiplier. In both chains, the traffic is so low that we are actually the bottom of this value. This means, that we are only charging `fee * 1e-9` of the weight portion, which is quite bad. This PR sets this value to `1/10` as a temporary means to figure out if we want the weight fee multiplier and this whole traffic adjustment or not.

2. **The average on initialize weight** is currently set to 10% of the block weight. This means that this portion cannot be used for the NPOS solution either (see: https://polkadot.network/polkadot-staking-an-update/). Looking at @shawntabrizi's tool: https://www.shawntabrizi.com/substrate-weight-graph/?endpoint=wss://rpc.polkadot.io/&start=2031851&end=2131851 Wee see that in a recent range the average is around `6B`, and the total weight is 2T, which means ~ 3/1000, so a real estimate would have been around 0.3%. We will still be conservative and set the value to 2.5%. 

image

### CODE REVIEW

**Comment by thiolliere** [comment]
about 1. What is the reason for minimum multiplier being 1/10 ? if blocks are far from full why shouldn't we lower the fees below this ? Anyway user can use Tip if someone else try to get the full block for cheap.
Also I thought validator get their reward mostly from staking.
anyway it is hard for me to tell about those economic stuff.

about the average on initialize weight looks good to me

File: runtime/common/src/lib.rs [hunk 1]
```diff
@@ -75,7 +75,7 @@ parameter_types! {
 	/// Minimum amount of the multiplier. This value cannot be too low. A test case should ensure
 	/// that combined with `AdjustmentVariable`, we can recover from the minimum.
 	/// See `multiplier_can_grow_from_zero`.
-	pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 1_000_000_000u128);
+	pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 10u128);
```
Comment (shawntabrizi) [inline runtime/common/src/lib.rs:78]:
```suggestion
	pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 1_000u128);
```

As I suggested in the research channel.

Some data:

```
Weight | Multiplier | Fee       
0     | FixedU128(1.000000000000000000) | 266,666,626
0     | FixedU128(0.100000000000000000) | 266,666,626
0     | FixedU128(0.010000000000000000) | 266,666,626
0     | FixedU128(0.001000000000000000) | 266,666,626
0     | FixedU128(0.000001000000000000) | 266,666,626
0     | FixedU128(0.000000001000000000) | 266,666,626
10    | FixedU128(1.000000000000000000) | 266,933,333,026
10    | FixedU128(0.100000000000000000) | 26,933,333,266
10    | FixedU128(0.010000000000000000) | 2,933,333,290
10    | FixedU128(0.001000000000000000) | 533,333,292
10    | FixedU128(0.000001000000000000) | 266,933,292
10    | FixedU128(0.000000001000000000) | 266,666,892
20    | FixedU128(1.000000000000000000) | 533,599,999,426
20    | FixedU128(0.100000000000000000) | 53,599,999,906
20    | FixedU128(0.010000000000000000) | 5,599,999,954
20    | FixedU128(0.001000000000000000) | 799,999,958
20    | FixedU128(0.000001000000000000) | 267,199,959
20    | FixedU128(0.000000001000000000) | 266,667,159
30    | FixedU128(1.000000000000000000) | 800,266,665,826
30    | FixedU128(0.100000000000000000) | 80,266,666,546
30    | FixedU128(0.010000000000000000) | 8,266,666,618
30    | FixedU128(0.001000000000000000) | 1,066,666,625
30    | FixedU128(0.000001000000000000) | 267,466,625
30    | FixedU128(0.000000001000000000) | 266,667,425
40    | FixedU128(1.000000000000000000) | 1,066,933,332,226
40    | FixedU128(0.100000000000000000) | 106,933,333,186
40    | FixedU128(0.010000000000000000) | 10,933,333,282
40    | FixedU128(0.001000000000000000) | 1,333,333,291
40    | FixedU128(0.000001000000000000) | 267,733,292
40    | FixedU128(0.000000001000000000) | 266,667,692
50    | FixedU128(1.000000000000000000) | 1,333,599,998,626
50    | FixedU128(0.100000000000000000) | 133,599,999,826
50    | FixedU128(0.010000000000000000) | 13,599,999,946
50    | FixedU128(0.001000000000000000) | 1,599,999,958
50    | FixedU128(0.000001000000000000) | 267,999,959
50    | FixedU128(0.000000001000000000) | 266,667,959
60    | FixedU128(1.000000000000000000) | 1,600,266,665,026
60    | FixedU128(0.100000000000000000) | 160,266,666,466
60    | FixedU128(0.010000000000000000) | 16,266,666,610
60    | FixedU128(0.001000000000000000) | 1,866,666,624
60    | FixedU128(0.000001000000000000) | 268,266,625
60    | FixedU128(0.000000001000000000) | 266,668,225
70    | FixedU128(1.000000000000000000) | 1,866,933,331,426
70    | FixedU128(0.100000000000000000) | 186,933,333,106
70    | FixedU128(0.010000000000000000) | 18,933,333,274
70    | FixedU128(0.001000000000000000) | 2,133,333,290
70    | FixedU128(0.000001000000000000) | 268,533,292
70    | FixedU128(0.000000001000000000) | 266,668,492
80    | FixedU128(1.000000000000000000) | 2,133,599,997,826
80    | FixedU128(0.100000000000000000) | 213,599,999,746
80    | FixedU128(0.010000000000000000) | 21,599,999,938
80    | FixedU128(0.001000000000000000) | 2,399,999,957
80    | FixedU128(0.000001000000000000) | 268,799,959
80    | FixedU128(0.000000001000000000) | 266,668,759
90    | FixedU128(1.000000000000000000) | 2,400,266,664,226
90    | FixedU128(0.100000000000000000) | 240,266,666,386
90    | FixedU128(0.010000000000000000) | 24,266,666,602
90    | FixedU128(0.001000000000000000) | 2,666,666,623
90    | FixedU128(0.000001000000000000) | 269,066,625
90    | FixedU128(0.000000001000000000) | 266,669,025
100   | FixedU128(1.000000000000000000) | 2,666,933,330,626
100   | FixedU128(0.100000000000000000) | 266,933,333,026
100   | FixedU128(0.010000000000000000) | 26,933,333,266
100   | FixedU128(0.001000000000000000) | 2,933,333,290
100   | FixedU128(0.000001000000000000) | 269,333,292
100   | FixedU128(0.000000001000000000) | 266,669,292
```

**Comment by kianenigma** [comment]
as decided offline, downgraded the PR to only update the on_initalize weight. This will help a bit with NPoS. Ready to merge.

**Comment by kianenigma** [comment]
bot merge

File: runtime/kusama/src/lib.rs [hunk 2]
```diff
@@ -1301,3 +1301,75 @@ sp_api::impl_runtime_apis! {
...
+	use frame_support::storage::StorageValue;
+	use sp_runtime::FixedPointNumber;
+	use frame_support::weights::GetDispatchInfo;
+	use codec::Encode;
+	use pallet_transaction_payment::Multiplier;
+	use separator::Separatable;
+
+
+	#[test]
+	#[ignore]
```
Comment (bkchr) [inline runtime/kusama/src/lib.rs:1324]: What is the purpose of these ignored tests?

**Review by bkchr — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

File: runtime/kusama/src/lib.rs (hunk 2 above)
Comment (kianenigma) [reply runtime/kusama/src/lib.rs:1324]: The other aspect of the PR that we decided to address later: it help print some stat about the fee stuff.

**Comment by kianenigma** [comment]
bot merge

**Pull request merged**